src/foam/dao/EasyDAO.js

```diff
@@ -777,6 +781,10 @@ model from which to test ServiceProvider ID (spid)`,
       class: 'Boolean',
       javaFactory: 'return getEnableInterfaceDecorators() && foam.nanos.crunch.lite.Capable.class.isAssignableFrom(getOf().getObjClass());'
     },
+    {
+      name: 'allowActionRequiredPuts',
+      class: 'Boolean'
```

Comment:
Can you add some documentation - on why someone would want to use this feature. 🙏 

From my confused understanding:  (Can use below to answer and use as documentation - Im not the only one not going to clearly see what Im looking at)
Doesn't a ucj/capableObj, in action_required always need to be put to a dao to save? Then we would want this feature to be on(allowActionRequiredPuts=true) only when we would not want to save any changes to an obj. But then, that isn't fully a feature of turning on/off the data save feature, or would it? Although we are turning on/off puts -through the wizard- we aren't considering any other possible status' or locations where a put is happening. This instead would be just one status change that would be used in a set of configurations?

Reply:
> Doesn't a ucj/capableObj, in action_required always need to be put to a dao to save?

- UCJ in action required: **yes**
- Capable in action required: **no**; default behavior (with and without this PR) is to NOT put the Capable in if it has required payloads in `ACTION_REQUIRED` - instead prompt the user with a wizard by sending a CapabilityIntercept.

> Then we would want this feature to be on(allowActionRequiredPuts=true) only when we would not want to save any changes to an obj.

The opposite, if I understood your question. The name begins with "allow" to indicate that it allows the put when a Capable has capabilities in `ACTION_REQUIRED`.